mscr-ui/src/common/components/side-navigation/side-navigation.styles.tsx

```diff
@@ -56,6 +59,7 @@ export const MscrSideNavigationLevel1 = styled(SideNavigationItem)`
 
 export const MscrSideNavigationLevel2 = styled(SideNavigationItem)`
   // Remove arrow icon from group buttons
+  margin-right: ${(props) => props.theme.suomifi.spacing.m + '!important'};
```

Comment:
Instead of !important, persistent styles should be overridden like this:
```
&& {
  margin-right: ${(props) => props.theme.suomifi.spacing.m};
}
```
And if needed, adding more &-characters to the && until the style gets overridden.

Reply:
I'll fix this.

Reply:
Great!